Observer/Config/Save.php

```diff
@@ -65,40 +65,58 @@ public function execute(Observer $observer)
             $this->cacheTypeList->cleanType(Config::TYPE_IDENTIFIER);
             $this->appConfig->reinit();
 
-            $scope = $scopes = null;
+            $scope = $scopes = \Magento\Framework\App\Config\ScopeConfigInterface::SCOPE_TYPE_DEFAULT;
```

Comment:
why do we need 2 different variables: "scope" and "scopes"? i do not see any different value in this function between the two - it looks redundant to me. Am I missing something?

Reply:
They are not always the same.

Reply:
please elaborate - where in the code can these 2 be different?